colour/io/luts/tests/test_sony_spi3d.py

```diff
@@ -215,7 +213,7 @@ def test_write_LUT_SonySPI3D(self):
 
                 tokens = line.split()
                 if len(tokens) == 6:
-                    indexes.append(parse_array(tokens[:3], DEFAULT_INT_DTYPE))
+                    indexes.append(as_float_array(tokens[:3]))
```

Comment:
Careful here also! The indexes/indices should be of type int, so you should use `as_int_array`:

```python
indexes.append(as_int_array(tokens[:3]))
```

Reply:
Noted